pkg/devicemapper/devmapper.go

```diff
@@ -264,14 +259,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-var dmLogger DevmapperLogger
-
-// LogInit initializes the logger for the device mapper library.
-func LogInit(logger DevmapperLogger) {
-	dmLogger = logger
-	LogWithErrnoInit()
-}
```

Comment:
I am again concerned about somebody using devicemapper package and using this interface and they will be broken now. 

I think rancher folks were using pkg/devicemapper.

This change does not seem fundamental to what you are trying to do. So why not first focus on the core change instead of removing existing interfaces of pkg/devicemapper/